src/System.IO.FileSystem/ref/System.IO.FileSystem.cs

```diff
@@ -251,8 +251,8 @@ namespace System.IO.Enumeration
     public ref struct FileSystemEntry
     {
         public ReadOnlySpan<char> Directory { get { throw null; } }
-        public string RootDirectory { get { throw null; } }
-        public string OriginalRootDirectory { get { throw null; } }
+        public ReadOnlySpan<char> RootDirectory { get { throw null; } }
```

Comment:
I'm surprised no tests had to change.  Are there not any that touch/manipulate these? e.g. comparing to null?

Reply:
Everything gets the data through `ToFullPath()`, etc.

Reply:
So... shouldn't we have tests that touch these directly?

Reply:
I'll add some direct hits to these from tests. The To* methods hit both of these properties so the are covered, just not directly.

Reply:
Great. Thanks.